nextflow.yaml

```diff
@@ -0,0 +1,65 @@
+package:
+  name: nextflow
+  version: 24.10.3
+  epoch: 0
+  description: A DSL for data-driven computational pipelines.
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - openjdk-21
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - openjdk-21
+      - openjdk-21-default-jvm
+  environment:
+    LANG: en_US.UTF-8
+    JAVA_HOME: /usr/lib/jvm/java-21-openjdk
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/nextflow-io/nextflow.git
+      expected-commit: 6183fdf9dbd9114f5ee86d11bc5e69cbd06501a6
+      tag: v${{package.version}}
+
+  - runs: |
+      export JAVA_HOME=/usr/lib/jvm/java-21-openjdk
+      mkdir -p ${{targets.destdir}}/usr/bin
+      mv nextflow ${{targets.destdir}}/usr/bin/nextflow
+
+subpackages:
+  - name: ${{package.name}}-compat
+    description: Compatibility package for Nextflow
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.subpkgdir}}
+          ln -sf /usr/bin/nextflow "${{targets.subpkgdir}}/nextflow"
+    test:
+      pipeline:
+        - name: Check symlink
+          runs: stat /nextflow
+
+update:
+  enabled: true
+  github:
+    identifier: nextflow-io/nextflow
+    strip-prefix: v
+
+test:
+  environment:
+    contents:
+      packages:
+        - openjdk-21
+        - openjdk-21-default-jvm
+        - curl
+    environment:
+      JAVA_HOME: /usr/lib/jvm/java-21-openjdk
+  pipeline:
+    - name: Test ant build
```

Comment:
nit: help and version tests and we can consider a --version check followed by grepping package version as well if binary exposes that.

Reply:
+1 lets see if we can expand the tests more